modules/ossm-about-adding-projects-using-smmr.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-create-mesh.adoc
+
+:_content-type: CONCEPT
+[id="ossm-about-adding-projects-using-smmr_{context}"]
+= About adding projects using the member roll resource
+
+Using the `ServiceMeshMemberRoll` resource is the simplest way to add a project to a service mesh. To add a project, specify the project name in the `spec.members` field of the `ServiceMeshMemberRoll` resource. The `ServiceMeshMemberRoll` resource specifies which projects are controlled by the `ServiceMeshControlPlane` resource. 
+
+[NOTE]
+====
+Adding projects using this method requires the user to have the `update servicemeshmemberrolls` and the `update pods` privileges in the project that is being added.
```

Comment:
I think will be good to add also here in the notes that the service mesh member roll need to be created before to start to adding projects using this method

Reply:
This is captured in the prerequisites of the task.